lib/ts/recipe/session/recipeImplementation.ts

```diff
@@ -262,7 +261,11 @@ export default function getRecipeInterface(
                         res,
                         "access",
                         response.accessToken.token,
-                        response.accessToken.expiry,
+                        // We set the expiration to 100 years, because we can't really access the expiration of the refresh token everywhere we are setting it.
+                        // This should be safe to do, since this is only the validity of the cookie (set here or on the frontend) but we check the expiration of the JWT anyway.
+                        // Even if the token is expired the presence of the token indicates that the user could have a valid refresh
+                        // Setting them to infinity would require special case handling on the frontend and just adding 10 years seems enough.
+                        Date.now() + 3153600000000,
```

Comment:
in the decision log, we wrote to change it to 100 years, but we didn't write why. The comment above says that we don't have access to the refresh token's lifetime all the time - but that's not true since the session information has the expiry time.

Reply:
We want to avoid querying the core, and that's why it's not available all the time.